mne/preprocessing/ica.py

```diff
@@ -249,11 +249,14 @@ class ICA(ContainsMixin):
               ii) additional noise reduction, it is a matter of current debate
               whether pre-ICA dimensionality reduction could decrease the
               reliability and stability of the ICA, at least for EEG data and
-              especially during preprocessing [5]_.
+              especially during preprocessing [5]_. (But see also [6]_ for a
+              possibly confounding effect of the different whitening/sphering
+              methods used in this paper (ZCA vs. PCA).)
               On the other hand, for rank-deficient data such as EEG data after
-              average reference or interpolation, it is recommended to reduce
-              the dimensionality by 1 for optimal ICA performance
-              (see the `EEGLAB wiki <eeglab_wiki_>`_).
+              average reference or interpolation, it is  recommended to reduce
+              the dimensionality (by 1 for  average reference and 1 for each
```

Comment:
two spaces after `for`. Good you added that, I though it Clemens included that earlier.

Reply:
... and before recommended ;)